invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LocationList/state/reducer.js

```diff
@@ -26,6 +33,28 @@ export default (state = initialState, action) => {
         error: action.payload,
         hasError: true,
       };
+    case DELETE_IS_LOADING:
+      return { ...state, isLoading: true };
+    case DELETE_SUCCESS:
+      return {
+        ...state,
+        isLoading: false,
+        data: {
+          hits: state.data.hits.filter(
```

Comment:
@ntarocco as it concerns this one, what happens here is, I manually remove the removed entry from the existing list. After delete I am not submitting a request to refetch them. 

My initial expectation was that when I delete, I would get back in the response the data of the deleted record, but my delete responses come back with empty data.

`{data: "", status: 204, statusText: "NO CONTENT", headers: {…}, config: {…}, …}`

Reply:
@topless I would leave the reducer logic simple with no logic and do this filtering in your action before and just pass the data in the payload. So this would end up just updating the data with what comes from the action. WDYT?

Reply:
@topless this still need to be moved to the action as discussed, no?

Reply:
@ntarocco this was removed completely, we are re-fetching with a small delay (the Karolina way) to ensure their deletion.